src/content_delegate.js

```diff
@@ -586,9 +595,34 @@ ContentDelegate.prototype.onDownloadAllSubmit = async function (event) {
   const browserSpecificFetch =
     navigator.userAgent.indexOf('Safari') + navigator.userAgent.indexOf('Chrome') < 0 ? content.fetch : window.fetch;
 
+  // load options
+  const options = await getItemsFromStorage('options');
+  // generate the filename
+  const pacerCaseId = event.data.id.match(/caseid\=\d*/)[0].replace(/caseid\=/, '');
```

Comment:
OK, this is actually why I'm here. `\=` is not a valid escape sequence (outside of a v-mode character class) so this should just be `/caseid=\d*/`. I realize this problem was introduced before and you're just moving the code, but it should still be corrected.

Reply:
You are correct! `\=` is not a valid escape sequence. Thank you for bringing this to our attention.

We can also use a capturing group and remove the replace() method.